### PR TITLE
Make it possible to customize variant label

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
Update to reference Kubernetes application config in order to customize variant label.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
Make it possible to customize the variant label of Kubernetes application
```

### CODE REVIEW

**Comment by nghialv** [comment]
Please add those fields to the doc.
https://pipecd.dev/docs/user-guide/configuration-reference/#kubernetes-application
https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs-dev/user-guide/configuration-reference.md

File: pkg/app/piped/executor/kubernetes/traffic.go [hunk 1]
```diff
@@ -111,7 +113,6 @@ func (e *deployExecutor) ensureTrafficRouting(ctx context.Context) model.StageSt
 		primaryPercent,
 		canaryPercent,
 		baselinePercent,
-		e.appCfg.TrafficRouting,
```
Comment (nghialv) [inline pkg/app/piped/executor/kubernetes/traffic.go:114]: Nice catch!

**Comment by nghialv** [comment]
Well done. Please update the doc and then
/lgtm

**Comment by nghialv** [comment]
Nice.
/lgtm

**Comment by knanao** [comment]
Is it better to change "Does this PR introduce a user-facing change?" ?
For instance, "Make it possible to customize variant label through Kubernetes application config"

**Comment by nghialv** [comment]
> Is it better to change "Does this PR introduce a user-facing change?" ? For instance, "Make it possible to customize variant label through Kubernetes application config"

Right. It should be. e.g.

> Make it possible to customize variant label of Kubernetes application

**Comment by knanao** [comment]
TYVM:D
Let me apply it!

**Comment by khanhtc1202** [comment]
You rock 🤘
/approve

**Pull request merged**